src/TraceEvent/Parsers/LinuxKernelEventParser.cs

```diff
@@ -162,6 +164,8 @@ protected internal override void Dispatch()
         }
         protected internal override void Validate()
         {
+            // TODO: write new Validate for this TraceEvent
```

Comment:
Looks great!  Is this TODO still applicable?  If I understand correctly, it can be removed.

Reply:
I'm actually not sure about this one.  I commented out the automatically generated validate method because the assert doesn't pass. The original one looks very confusing to me, and I wonder if we need to write new ``` Validate()``` for this traceevent. If so, my plan is to leave it to the next PR.

Reply:
I think you can get rid of this TODO - as you pointed out to me offline the data in the event isn't being parsed properly.  That would need to be fixed before a new Validate method can be written.

Reply:
Seems more complicated than I thought. I'll resolve it for now and we can talk about this later offline.

Reply:
Sounds good.  Is this ready for merge?

Reply:
yes I'll merge it

Reply:
Yes, just realized I can't merge it.